packages/rebox/web/src/build-web/get-config.ts

```diff
@@ -65,6 +65,16 @@ export const getConfig = (entryPointPath: string, outputPath: string, htmlTempla
           },
         ],
       },
+      {
+        test: /\.png$/,
+        exclude: /node_modules/,
+        loader: 'file-loader?name=images/[name].[ext]',
```

Comment:
let's make it `images/[name].[hash].[ext]` like the `js/[name].[chunkhash].js` above